examples/gke-security-best-practices/create-tfvars.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# set some variables:
+export GOOGLE_PROJECT=$(gcloud config get-value project)
+export MY_PUBLIC_IPV4=$(curl https://api.ipify.org)
+
+# create our variables file:
+cat - > terraform.tfvars <<EOF
+
+# project properties
+project="${GOOGLE_PROJECT}"
+name="${GOOGLE_PROJECT}"
+region="us-west2"
+
+# custom network properties
+network_name="${GOOGLE_PROJECT}-net"
+ip_cidr_range="172.16.0.0/28"
```

Comment:
Don't put defaults into the .tfvars file - they should simply be defaults on `var` definition in Terraform.

Reply:
agreed will update to reflect